packages/components/package.json

```diff
@@ -21,17 +21,20 @@
     "license": "MIT",
     "dependencies": {
```

Comment:
Why do we need this file at all? I would have hoped we could just have package.json files in each subcomponent's files.